### PR TITLE
Javadoc updates 1

### PR DESCRIPTION
**Pull request by shoatman**
Updated javadoc and small accessor changes.

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/AcquireTokenSilentParameters.java [hunk 1]
```diff
@@ -50,10 +53,23 @@ public Boolean getForceRefresh() {
...
         private Boolean mForceRefresh;
 
+
+        /**
+         * <p>
+         *     NOTE: In general you should not use this method.  MSAL will refresh the token automatically using
+         *     it's own heuristic.
+         * </p>
+         * @param force Tells MSAL to refresh the token as part of this request
+         * @return
```
Comment (hamiltonha) [inline msal/src/main/java/com/microsoft/identity/client/AcquireTokenSilentParameters.java]: @return the builder?

File: msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java [hunk 2]
```diff
@@ -213,6 +221,15 @@ private void checkCurrentAccountNotifyCallback(@NonNull final CurrentAccountCall
...
 
+    /**
+     * @param activity Non-null {@link Activity} that is used as the parent activity for launching the {@link AuthenticationActivity}.
+     * @param scopes   The non-null array of scopes to be consented to during sign in.
+     *                 MSAL always sends the scopes 'openid profile offline_access'.  Do not include any of these scopes in the scope parameter.
+     *                 The access token returned is for MS Graph and will allow you to query for additional information about the signed in account.
+     * @param callback {@link AuthenticationCallback} that is used to send the result back. The success result will be
+     *                 sent back via {@link AuthenticationCallback#onSuccess(IAuthenticationResult)}.
+     *                 Failure case will be sent back via {
+     */
```
Comment (hamiltonha) [inline msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java]: failure case will be sent back via what?

Comment (shoatman) [reply msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java]: Fixed. Thanks

File: msal/src/main/java/com/microsoft/identity/client/AcquireTokenSilentParameters.java [hunk 3]
```diff
@@ -50,10 +53,23 @@ public Boolean getForceRefresh() {
...
+     */
     public static class Builder extends TokenParameters.Builder<AcquireTokenSilentParameters.Builder> {
 
         private Boolean mForceRefresh;
 
+
+        /**
+         * <p>
+         *     NOTE: In general you should not use this method.  MSAL will refresh the token automatically using
+         *     it's own heuristic.
```
Comment (rpdome) [inline msal/src/main/java/com/microsoft/identity/client/AcquireTokenSilentParameters.java]: nit: typo

File: msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java [hunk 4]
```diff
@@ -88,8 +88,7 @@ void acquireToken(@NonNull final Activity activity,
 
     /**
      * Returns the PublicClientConfiguration for this instance of PublicClientApplication
-     * Configuration is based on the defaults established for MSAl and can be overridden by creating the
-     * PublicClientApplication using {@link PublicClientApplication#PublicClientApplication(Context, PublicClientApplicationConfiguration)}
+     * Configuration is based on the defaults established for MSAl and can be overridden using constructor methods that accept configuration
```
Comment (rpdome) [inline msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java]: nit: MSAL

File: msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java [hunk 5]
```diff
@@ -311,11 +303,8 @@ public static void create(@NonNull final Context context,
      *                             and <a href="https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki">MSAL Github Wiki</a>
      *                             </p>
      * @return An instance of IPublicClientApplication.
-     * @throws IllegalStateException if this function is invoked on the main thread.
```
Comment (rpdome) [inline msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java]: Should we keep this? (or explicitly call out that this function can be invoked on a bg thread only)

Comment (rpdome) [reply msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java]: here and below

File: msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java [hunk 6]
```diff
@@ -89,6 +89,9 @@ private void initializeSharedPreferenceFileManager(@NonNull final Context contex
                 new StorageHelper(context));
     }
 
+    /**
```
Comment (rpdome) [inline msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java]: Javadoc comments in this file already exists in its interface file(s). If we need to update, shouldn't we update it there instead?

File: msal/src/main/java/com/microsoft/identity/client/TenantProfile.java [hunk 7]
```diff
@@ -28,13 +28,19 @@
 import com.microsoft.identity.common.internal.providers.microsoft.MicrosoftIdToken;
 import com.microsoft.identity.common.internal.providers.oauth2.IDToken;
 
+/**
+ * TenantProfile implements the {@link com.microsoft.identity.client.ITenantProfile ITenantProfile} interface.
+ */
 public class TenantProfile extends Account implements ITenantProfile {
 
-    public TenantProfile(@Nullable final String clientInfo,
+    TenantProfile(@Nullable final String clientInfo,
```
Comment (kreedula) [inline msal/src/main/java/com/microsoft/identity/client/TenantProfile.java]: nit : needs a formatter run

**Comment by kreedula** [comment]
Reminder to myself and @shoatman @hamiltonha these needs to be cherry-picked to release/1.0.0 branch from dev once merged.

**Pull request closed without merging**